test/d-b-n_smoke/arrays.html

```diff
@@ -248,6 +248,26 @@ <h4>
                         }, 100);
                     });
                 });
+
+                describe('Undesired but expected. Polymer does not reflect deleting props', function () {
+                    // see: https://github.com/Polymer/polymer/issues/2565
```

Comment:
I think we could console.log the issue link so the reason would be more visible for person running test suite.